composer.json

```diff
@@ -16,7 +16,7 @@
     "guzzlehttp/guzzle": "^6.4 || ^7.0",
     "psr/http-client": "^1.0",
     "psr/http-message": "^1.0",
-    "psr/log": "^1.1",
+    "psr/log": "^3.0",
```

Comment:
We will update this, when PHP 7.4 will be in "End of life".
https://www.php.net/supported-versions.php